flow/flow.go

```diff
@@ -1080,6 +1190,13 @@ func segmentProcess(parameters interface{}, inIndex []int32, stopper [2]chan int
 							count := FillSliceFromMask(InputMbufs, &def[st].mask, OutputMbufs[0])
 							safeEnqueue(OUT[answers[0]][inIndex[q]], OutputMbufs[0], uint(count))
 							currentSpeed.Packets += uint64(count)
+
+							if countersEnabled {
+								// FIXME: inaccurate statistics because
```

Comment:
I suppose that one FIXME (TODO is everywhere in other places) is enough, I propose not to put it in all places.